### PR TITLE
IOMMU boot test

### PR DESCRIPTION
**Pull request by emanchado**
Add a IOMMU boot test with different parameters. It works on RHEL8, RHEL7, and Fedora 30 at least. It will have to be adapted to work on older RHEL releases due to the use of `journalctl`, though.

### CODE REVIEW

**Comment by emanchado** [comment]
@cki-bot test this stuff, please!

File: iommu/boot/runtest.sh [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    dmesgErrors
+else
+    if [[ $cpuvendor = "GenuineIntel" ]]; then
+	bootOptions $DefaultBootOptionsIntel
+	dmesgErrors
+    elif [[ $cpuvendor = "AuthenticAMD" ]]; then
+       	bootOptions $DefaultBootOptionsAMD
+	dmesgErrors
+    else
+	report_result "${TEST}/nonAMDorIntelProcessor" "FAIL" 0
```
Comment (veruu) [inline iommu/boot/runtest.sh]: Should this be SKIP instead, as it's not the patch developer's fault that the test is running on unsupported machine?

**Comment by veruu** [comment]
@emanchado just FYI, if the test wasn't already added to kpet-db, the generated XML won't contain this test and it won't run. /me is not on VPN to check that

File: iommu/boot/runtest.sh (hunk 1 above)
Comment (emanchado) [reply iommu/boot/runtest.sh]: Sounds good. That was in the original test, and I didn't even know `SKIP` was a thing in this harness :sweat_smile: 

Which reminds me: how much are we supposed to review/change the tests when copying them?

**Pull request merged**